src/terrain/util.clj

```diff
@@ -13,6 +15,10 @@
         (vector? p) p
         :else       [p]))
 
+(defmethod compojure.api.meta/restructure-param :description-file
```

Comment:
🏆 

Maybe this should eventually be defined in the `common-swagger-api` lib, if that's possible? So markdown file docs could be shared between common `terrain` and `apps` service endpoints, for example? Not sure yet how the resource path for a doc in the lib would look 🤔

Reply:
I'm pretty sure it'll work without modification if the file happens to be in a different JAR file. We'll just have to watch out for file name collisions.

Reply:
I really like the idea of putting this in `common-swagger-api`

Reply:
> We'll just have to watch out for file name collisions.

Good point. I guess we could keep the docs in subfolders that mimic the endpoint or Swagger tag. For example: `resources/docs/apps/app-metadata/get-app-metatada.md` or something like that.

Reply:
Yeah, that's a good idea.